features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -1051,6 +1051,12 @@
   "console.ui.hiddenConnectionTemplates": [ "hypr-idp", "swe-idp" ],
   "console.ui.google_one_tap_enabled_tenants": [],
   "console.ui.show_app_switch_button": true,
+  "console.ui.legacy_features": {
```

Comment:
Lets use `dot` notation. Coz sometime nested objects interfere when trying to override through TOML.

ex:
```
"console.ui.legacy_features.applicationOIDCSubjectIdentifierEnabled": false
```